ampproject/amphtml
# 🐛amp-ima-video: show play/pause button in ad controls

Closes #22952

# Description:

- No longer hide play/pause button in ad controls
- Un-skips all tests
- Updates tests to check new play/pause functionality
- Improves ads manager mocking in tests

# Steps to test:

1. Use `gulp` command to run examples locally
2. Visit http://localhost:8000/examples/ima-video.amp.html on desktop or using a mobile emulator
3. Check the 4 test cases described in the following screenshots
    * desktop with big play button (video 3)
    * desktop autoplay (video 4)
    * mobile with big play button (video 3)
    * mobile with autoplay (video 4)

# Known Issues

Default ad countdown timer is not hidden while ads are playing on mobile.  This has not been changed / fixed in this commit.

# Screenshots:

## Desktop

### Manual play

![desktop-manual](https://user-images.githubusercontent.com/4807680/62382892-34b8a400-b514-11e9-8559-dc99cd20fcf5.gif)

### Autoplay

![desktop-autoplay](https://user-images.githubusercontent.com/4807680/62383120-be687180-b514-11e9-92ff-0609b165c0aa.gif)


## Mobile

### Manual play

![mobile-manual](https://user-images.githubusercontent.com/4807680/62382122-8eb86a00-b512-11e9-85dc-f8e02b45c3d6.gif)

### Autoplay

![mobile-autoplay](https://user-images.githubusercontent.com/4807680/62382549-65e4a480-b513-11e9-93e7-b2f593a9dd42.gif)
